modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Adding permissions for cluster configuration
+
+You can grant permissions for an Argo CD instance to manage cluster configuration. Create a cluster role with additional permissions and then create a new cluster role binding to associate the cluster role with a service account. 
+
+.Procedure
+
+. Log in to the {product-title} web console as an admin.
+. In the wev console, select **User Management** -> **Roles** -> **Create Role**. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
```

Comment:
```
In the web console,
```

Reply:
@rolfedh  Why are you leaving new review comments on a PR that was merged almost eight weeks ago?

Reply:
@JStickler As part of my review to be granted merge rights, I received some feedback on PRs that I created or reviewed recently. Some of these are issues I missed, others are guidelines I wasn't aware of. I'm passing along some of those items to folks whose PRs I reviewed. I also messaged the PR owners directly in Slack to let them know that this is what I was doing.